README.md

```diff
@@ -50,7 +50,11 @@ Please follow the steps mentioned below to build the WSO2 Streaming Integrator f
 
 ## Getting Started
 
-Get started with Streaming integrator in a few minutes by following [Streaming Integrator Quick Start Guide](https://docs.wso2.com/display/SP4xx/Quick+Start+Guide)
+Please refer the below guides to get you self started with the Streaming Integrator
```

Comment:
"get you self" needs to be corrected to "get yourself"

Reply:
Corrected through below commit